src/components/OfframpSummaryDialog/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { ArrowDownIcon } from '@heroicons/react/20/solid';
+import { FC } from 'preact/compat';
+import Big from 'big.js';
+
+import { InputTokenDetails, OutputTokenDetails } from '../../constants/tokenConfig';
+import { UseTokenOutAmountResult } from '../../hooks/nabla/useTokenAmountOut';
+import { useGetAssetIcon } from '../../hooks/useGetAssetIcon';
+import { useOfframpFees } from '../../hooks/useOfframpFees';
+import { useNetwork } from '../../contexts/network';
+import { Networks } from '../../helpers/networks';
+
+import { ExchangeRate } from '../ExchangeRate';
+import { NetworkIcon } from '../NetworkIcon';
+import { Dialog } from '../Dialog';
+
+interface AssetDisplayProps {
+  amount: string;
+  symbol: string;
+  iconSrc: string;
+  iconAlt: string;
+}
+
+const AssetDisplay = ({ amount, symbol, iconSrc, iconAlt }: AssetDisplayProps) => (
+  <div className="flex items-center justify-between w-full">
+    <span className="text-lg font-bold">
+      {amount} {symbol}
+    </span>
+    <img src={iconSrc} alt={iconAlt} className="w-8 h-8" />
+  </div>
+);
+
+interface FeeDetailsProps {
+  network: Networks;
+  feesCost: string;
+  fiatSymbol: string;
+  tokenOutAmount: UseTokenOutAmountResult;
+  fromToken: InputTokenDetails;
+  anchorUrl?: string;
+}
+
+const FeeDetails = ({ network, feesCost, fiatSymbol, tokenOutAmount, fromToken, anchorUrl }: FeeDetailsProps) => (
+  <section className="mt-6">
+    <div className="flex justify-between mb-2">
+      <p>Offramp fees</p>
```

Comment:
We should add in the brackets how this offramp fee is comprised. 
```suggestion
      <p>Offramp fee ({`${outputToken.offrampFeeBasisPoints}%` + outputToken.offrampFeesFixedComponent > 0 ? ` + {outputToken.offrampFeesFixedComponent}` :  ""})</p>
```